tox.ini

```diff
@@ -196,3 +196,11 @@ per-file-ignores =
 
     # F401 'foo' imported but unused
     motor/__init__.py: F401
+
+[testenv:typecheck-mypy]
+description = run mypy to typecheck
+deps =
+    mypy==1.2.0
+    typing_extensions
+commands =
+    mypy --install-types --non-interactive motor/ --disallow-untyped-defs --follow-imports=skip --exclude 'motor/aiohttp/*' --exclude 'motor/frameworks/*' --exclude 'motor/metaprogramming.py' --exclude 'motor/motor_gridfs.py' --exclude 'motor/web.py'
```

Comment:
We should type gridfs for parity.

Reply:
Does `gridfs` have public APIs? If not then is it worth the extra time to add types to it?

Reply:
Yes motor_gridfs has many public apis.

Reply:
Got it! I'll add types for `gridfs` then